extensions/Worker.Extensions.Http.AspNetCore/src/Coordinator/ContextReference.cs

```diff
@@ -59,7 +62,7 @@ internal void CompleteFunction()
 
             if (_httpContextValueSource.Task.IsCompleted)
             {
-                if (_httpContextValueSource.Task.IsCanceled || _token.IsCancellationRequested)
+                if (_httpContextValueSource.Task.IsCanceled || _invocationToken.IsCancellationRequested || _token.IsCancellationRequested)
```

Comment:
No piece of code will ever cancel `_token`, so it could be removed. My other comment also touches on how `CancellationToken` may not exactly be the right type.